kustomize/data-plane/deployDataPlane.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+source common_tools.sh
+
+oc get project istio-system
+status=$?
+if [ $status -ne 0 ]; then
+  echo "The istio-system project does not exist, creating"
+  oc new-project istio-system
```

Comment:
@winklerm Is there any reason why the creation of the namespace cannot be in the `kustomize` resources so that's created declaratively rather than imperatively?

Longer-term, what should be creating this `namespace`? The actual installation of the Openshift ServiceMesh operator?

Reply:
This is a difference in the upstream istio operator and the servicemesh operator that I have run into: The ServiceMesh operator documentation [1] says the namespace should be created manually (while istioctl will create it automatically).

Good point with creating the namespace declaratively, I just followed the docs and did not realize that, I will change this, thanks!

[1] https://docs.openshift.com/container-platform/4.10/service_mesh/v2x/ossm-create-smcp.html#ossm-control-plane-deploy-cli_ossm-create-smcp

Reply:
@winklerm Yeah let's see if it will work declaratively in our `kustomize` resources. Thanks for investigating.

cheers,
Rob